### PR TITLE
Update supported version of Ubuntu.

### PR DESCRIPTION
**Pull request by razum0v**

### CODE REVIEW

**Comment by Aokromes** [comment]
IMHO it can be safely increased to 18.04.

19th April: Final Freeze
26th April: Stable Ubuntu 18.04 LTS release

**Comment by john-preston** [comment]
@Aokromes But is it working there fine? %)

**Comment by Aokromes** [comment]
@john-preston i will download 18.04 and i will tell you later :)

**Comment by Aokromes** [comment]
https://github.com/telegramdesktop/tdesktop/issues/4573 well this guy have it working (idk if it's because windows 10 RS5 or vmware or ubuntu 18.04 but i am unable to run it on VM)

**Comment by Aokromes** [comment]
Confirmed Telegram desktop working fine on ubuntu 18.04 LTS.

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ The source code is published under GPLv3 with OpenSSL exception, the license is
 * Windows XP - Windows 10 (**not** RT)
 * Mac OS X 10.8 - Mac OS X 10.11
 * Mac OS X 10.6 - Mac OS X 10.7 (separate build)
-* Ubuntu 12.04 - Ubuntu 16.04
+* Ubuntu 12.04 - Ubuntu 17.10
```
Comment (stek29) [inline README.md]: according to @Aokromes 18.04 should be here

Comment (razum0v) [reply README.md]: @stek29, changed

**Comment by stek29** [comment]
Thanks!

**Pull request merged**